src/rules/at-if-no-null/README.md

```diff
@@ -0,0 +1,44 @@
+# at-if-no-null
+
+Check for equality to null is unnecessarily explicit since `null` is falsey in Sass.
+
+```scss
+a {
+    @if ($x == null) {}
```

Comment:
I'd like to discourage the use of parentheses with `@if` as well, so maybe these examples should omit them too?